### ISSUE TITLE
Borger.dk theme: Navigation "current" color does not conform with WCAG Success Criterion 1.4.3: Contrast (Minimum)

### ISSUE THREAD

**Issue by christianknu** [detfaellesdesignsystem/dkfds-components#116]
https://github.com/detfaellesdesignsystem/dkfds-components/blob/b1a53e6e55087a5f84b3afe86c6b8c0fb28782d6/dist/scss/components/_sidenav.scss#L138

The gray color used for indicating the current page in the sidenavigation does not conform with the _Success Criterion 1.4.3: Contrast (Minimum)_, stating that contrast ratio should be 4.5:1 at a minimum.

**Comment by detfaellesdesignsystem** [comment]
Hi @christianknu,

As we talked about earlier over teams we are currently waiting for more information on this issue.

Please update the issue when you have more information by either commenting or closing the issue entirely if you discover it's not relevant to your solution now.

Best regards,
Kristina Jensen
Det Fælles Designsystem

**Comment by christianknu** [comment]
After discussing this issue with Kristina and the customer, we have concluded, that the criterion doesn't apply to this scenario, hence I'm closing the issue.
